src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageAnalyzerResultProvider.cs

```diff
@@ -16,14 +16,17 @@ namespace Microsoft.DotNet.UpgradeAssistant.Steps.Packages
 {
     public class PackageAnalyzerResultProvider : IAnalyzeResultProvider
     {
-        private const string Id = "UA101";
+        private const string RuleId = "UA101";
+        private const string RuleName = "SomeRuleName";
+        private const string FullDescription = "Some full description";
+        private readonly Uri _helpUri = new("https://docs.microsoft.com/en-us/dotnet/core/porting/upgrade-assistant-overview");
```

Comment:
This is one example where we could create a specific `RuleName`, `FullDescription`, and `HelpUri`.
This will make the SARIF more complete.

Reply:
It works in the scenario of PackageAnalysis but likely won't for DiagnosticAnalysis as the the Rule Name is taken from the Diagnostic Analyzers.